src/reference/asciidoc/aggregator.adoc

```diff
@@ -117,11 +117,15 @@ NOTE: In the interest of code simplicity and promoting best practices such as lo
 Starting with version 5.1, after processing message group, an `AbstractCorrelatingMessageHandler` performs a `MessageBuilder.popSequenceDetails()` message headers modification for the proper splitter-aggregator scenario with several nested levels.
 It is done only if the message group release result is not a message or collection of messages.
 In that case a target `MessageGroupProcessor` is responsible for the `MessageBuilder.popSequenceDetails()` call while building those messages.
-Strting with version 5.3 if sequenceDetails of the outputMessage matches with first message of group, `MessageBuilder.popSequenceDetails()` will be performed.
+
+Additionally, Starting with version 5.3, if `MessageGroupProcessor` returns a Message, `MessageBuilder.popSequenceDetails()` will be performed on the `outputMessage`  if the sequenceDetails matches with first message of group.
```

Comment:
And why `sequenceDetails` is not a code any more?
It can be a plain text, but make it like that then!
I mean like this `sequence details`. Or bring it back to the code wrapping.